pkg/controller/logstorage/logstorage_controller.go

```diff
@@ -540,6 +540,13 @@ func (r *ReconcileLogStorage) Reconcile(request reconcile.Request) (reconcile.Re
 			r.status.SetDegraded("Waiting for curator secrets to become available", "")
 			return reconcile.Result{}, nil
 		}
+
+		// ES should be in ready phase when execution reaches here, apply ILM polices
+		if err = r.esClient.SetILMPolicies(r.client, ctx, ls); err != nil {
+			log.Info("Waiting for ES ILM policies and templates to get created")
```

Comment:
I think log.error could be a better choice, that way you can log the err as well.